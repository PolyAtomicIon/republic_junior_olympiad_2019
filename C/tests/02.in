16 16
265 331 718 753 569 1000 839 187 672 923 144 565 324 1000 533 483 