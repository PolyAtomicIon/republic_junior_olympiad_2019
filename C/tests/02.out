265 4571 42054 273497 1407501 6089916 23003423 77819959 240158550 685513385 829361377 603099199 996948180 86703369 908384937 774761780 